recipes_source/android_native_app_with_custom_op.rst

```diff
@@ -0,0 +1,732 @@
+Making Native Android Application that uses PyTorch prebuilt libraries
+======================================================================
+
+**Author**: `Ivan Kobzarev <https://github.com/IvanKobzarev>`_
+
+In this recipe, you will learn:
+
+ - How to make an Android Application that uses LibTorch API from native code (C++).
+
+ - How to use within this application TorchScript models with custom operators.
+
+Setup
+=====
```

Comment:
Nit: Recommend not to use the same header level === for sub-headers as is currently used for the title.